WeWatch/WeWatch/Custom TabBar/TabBar.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TabBar.swift
+//  WeWatch
+//
+//  Created by Anton on 30/12/2024.
+//
+
+import SwiftUI
+
+private struct TabBar: View {
+    
+    @StateObject private var viewModel: TabBarViewModel = .init()
+
+    internal var body: some View {
+        ZStack {
+            viewModel.selectedTab.activeView()
```

Comment:
Use a switch statement to determine the views based on `selectedTab: TabViewType` from the `ViewModel`. For example:
```
switch viewModel.selectedTab {
case .bookmark:
    BookmarkView()
// Add the remaining cases here
}
```
To avoid using `AnyView`, create a separate function that returns `some View` and mark it with `@ViewBuilder`. This approach improves performance and simplifies your code. You can learn more about `@ViewBuilder` here: [@ViewBuilder in SwiftUI](https://www.avanderlee.com/swiftui/viewbuilder/).